docs/dxp/7.x/en/installation-and-upgrades/setting-up-liferay-dxp/clustering-for-high-availability/configuring-unicast-over-tcp.md

```diff
@@ -150,12 +150,13 @@ The following steps use Unicast over TCPPing to demonstrate the approach.
 
     For example, your first two nodes might assign these bind ports:
 
+    <!-- Unfortunately Sphinx does not play nice with this table and it messes up the rendering of the text below it.
     | Node   | Properties File     | Port   |
     | :----- | :------------------ | :----- |
     | Node 1 | `tcp-control.xml`   | `7800` |
     | Node 1 | `tcp-transport.xml` | `7801` |
     | Node 2 | `tcp-control.xml`   | `7802` |
-    | Node 2 | `tcp-transport.xml` | `7803` |
+    | Node 2 | `tcp-transport.xml` | `7803` | -->
```

Comment:
Hmm. I'll look at using the RST format.

Reply:
Let me know if that works. I believe its a limitation of https://pypi.org/project/sphinx-markdown-tables/ - where the tables don't render correctly when you're trying to indent/nest them in a list.

Reply:
We have other areas that would benefit from being able to nest/indent a table as well.